pkg/cloud/ovirt/machine/actuator.go

```diff
@@ -555,13 +555,14 @@ func (actuator *OvirtActuator) autoPinningSupported(machine *machinev1.Machine,
 // Returns: nil or error
 func validateAutPinningPolicyValue(autopinningpolicy string) error {
 	switch autopinningpolicy {
-	case "disabled", "existing", "adjust":
+	case "disabled", "existing", "adjust", "none", "pin", "resize_and_pin":
```

Comment:
This seems to be a duplication between multiple libraries. Please consider adding these as constants to https://github.com/janoszen/goVirt. We are moving this lib to the oVirt org and hopefully use it as a backend anyway.

Reply:
For now it's ok, when we will port to goOvirt in the provider we will port that part as well :)